test/framework/kubernetesversions/versions.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetesversions
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/blang/semver"
+)
+
+const (
+	ciVersionURL     = "https://dl.k8s.io/ci/latest.txt"
+	stableVersionURL = "https://storage.googleapis.com/kubernetes-release/release/stable-%d.%d.txt"
+	tagPrefix        = "v"
+)
+
+// LatestCIRelease fetches the latest main branch Kubernetes version
+func LatestCIRelease() (string, error) {
```

Comment:
How about adding LatestRCRelease or latest release that is not an alpha release? I am thinking there is not much of a point in testing an alpha release. WDYT?

Reply:
The CI release testing is used for CAPI providers which are Kubernetes release informing, such as CAPA.